en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
-- Subtracting a numerical variable (`secondNumber`) from another numerical variable (`firstNumber`).
+    ```ballerina
+    2022 - yearOfBirth;
+    ```
+ 
+- Division
 
-  ```ballerina
-  firstNumber - secondNumber
-  ```
+    You can use the `/` operator for number division. 
 
-### Number Division
+    ###### Example:
 
-You can use the `/` operator for number division. The number values can come from a literal or a variable.
+    To divide an integer variable `totalNumberOfStudents` from integer literal `5`, you can use the following expression:
 
-#### Examples
+    ```ballerina
+    totalNumberOfStudents / 5; 
+    ```
+ 
+- Multiplication
 
-- Dividing integer literals.
+    You can use the `*` operator for number multiplication. 
 
-  ```ballerina
-  20 / 10 
-  ```
-  
-- Dividing decimal literals.
+    ###### Example:
 
-  ```ballerina
-  1.84 / 0.56  
-  ```
-  
-- Dividing a numerical variable (`firstNumber`) by a literal.
+    To multiply two float constants `60` by `5`, you can use the following expression:
 
-  ```ballerina
-  firstNumber / 100
-  ```
-  
-- Dividing a numerical variable (`firstNumber`) by another numerical variable (`secondNumber`).
+    ```ballerina
+    60 * 5;
+    ```
 
-  ```ballerina
-  firstNumber / secondNumber
-  ```
+- Modulo 
 
-### Number Multiplication
+    You can use the  `%` operator to get the remainder of a division. 
 
-You can use the `*` operator for number multiplication. The number values can come from a literal or a variable.
+    ###### Example:
 
-#### Examples
+    To get the remainder of the division between an integer variable `n` and an integer literal `3`, you can use the following expression:
+  
+    ```ballerina
+    (n % 3) > 2
+    ```
 
-- Multiplying integer literals.
+### Value/type operators
 
-  ```ballerina
-  10 * 20
-  ```
-  
-- Multiplying decimal literals.
+- Equality 
 
-  ```ballerina
-  0.56 * 1.84
-  ```
-  
-- Multiplying a numerical variable (`firstNumber`) by a literal.
+    You can use the equality operators to check the equality of [Ballerina's](https://ballerina.io/) basic type literals. These values can come from a literal or a variable. **Note** that the resulting value from these operators is always of `boolean` type.   
+    
+    Ballerina provides the following equality operators: 
+    
+    - `==`: For deep value equality of `anydata` typed values.
+    - `===`: For reference equality. 
+    - `!= `and `!==`: For negations.
 
-  ```ballerina
-  firstNumber * 100
-  ```
-  
-- Multiplying a numerical variable (`firstNumber`) by another numerical variable (`secondNumber`).
+    ###### Examples:
 
-  ```ballerina
-  firstNumber * secondNumber
-  ```
+    To compare an integer literal `10` with an integer variable `firstNumber`, you can use the following expression:
+    
+    ```ballerina
+    10 == firstNumber
+    ```
 
-### Modulo Operation
+    To compare a string variable `username`  with  a string literal `admin`, use the following expression:  
```

Comment:
Suggestion: "To compare a string variable (e.g., `username`) with a string literal (e.g., `admin`), use the following expression."